Remarkable-Donkeys/bangazonWebApp
# Mw inital migration

**The Name of this branch is**
mw-initialMigration


* **What kind of change does this PR introduce?** (Bug Fix, Feature, Document Update...)
Feature


* **What Ticket is associated with this PR**
N/A


* **What is the new behavior?**  (if this is a feature change)
User can register with their information,

* **List all core functionality**
Users can now register their bangazon account which is authenticated using identity framework. Additional information is required at registration, such as address and phone number. 

* **Other information / Steps to Test**:
1. `git fetch --all`
1. `git checkout mw-initialMigration`
1. run the webapp in the browser.
1. register a new user.
1. confirm the user data is stored in the database.


***At least one person needs to approve pull request before merged to master. Indicate if you think more than one person should test the feature***
